.github/workflows/bf-review.yml

```diff
@@ -45,7 +46,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        service: ["bf-bknd", "bf-appl"]
+        service: ["bf-bknd", "bf-appl", "bf-hand"]
```

Comment:
the name must be `bf-hdlr` because that is reserved namespace name, otherwise it does not work

Reply:
OK thanks, fixed!